### ISSUE TITLE
Add url or command line passed under banner while scan progress is displaying

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#50]
```
It would be _really_ helpful to display the command used to launch the scan
or at least the url getting scanned while the progress screen is up - maybe
under the skipfish version banner.

```

Original issue reported on code.google.com by `jtfalg...@gmail.com` on 1 Apr 2010 at 7:45

**Comment by GoogleCodeExporter** [comment]
```
It's actually sort of difficult because multiple host names or paths may be 
within the 
scope for an audit. I guess we could use the first one?
```

Original comment by `lcam...@gmail.com` on 1 Apr 2010 at 7:52
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Would it be easier just to echo the command and options used to launch the 
binary?

Or ..... maybe an option for some type of label/tag/title? 

I launched several scans in different windows and got confused as to which one 
was where.
```

Original comment by `jtfalg...@gmail.com` on 1 Apr 2010 at 8:56

**Comment by GoogleCodeExporter** [comment]
```
Well, options may span more than one line and add clutter. Audit title is 
actually a 
nice idea (that or output directory).


```

Original comment by `lcam...@gmail.com` on 1 Apr 2010 at 9:05

**Comment by GoogleCodeExporter** [comment]
```
Output directory would work too!!!

Thanks
```

Original comment by `jtfalg...@gmail.com` on 1 Apr 2010 at 9:32

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in 1.28
```

Original comment by `lcam...@gmail.com` on 2 Apr 2010 at 8:47
- Changed state: **Fixed**
